packages/accounts-controller/package.json

```diff
@@ -67,7 +67,7 @@
   },
   "peerDependencies": {
     "@metamask/keyring-controller": "^16.0.0",
-    "@metamask/snaps-controllers": "^7.0.1"
+    "@metamask/snaps-controllers": "^8.1.0"
```

Comment:
Shoudn't this be `8.1.1`? Or you wanted to allow any `^8.1.0`?

Reply:
No, youre right, it should be 8.1.1